src/System.Runtime.CompilerServices.Unsafe/src/System.Runtime.CompilerServices.Unsafe.ilproj

```diff
@@ -14,20 +14,28 @@
     <Reference Include="System.Runtime" />
   </ItemGroup>
 
-  <Target Name="AddResourcesFileToIlasm" BeforeTargets="CoreCompile" Condition="'$(OS)'=='Windows_NT'">
-    <MakeDir Directories="$(IntermediateOutputPath)Disassembly" />
-
+  <Target Name="AddResourcesFileToIlasm" 
+          BeforeTargets="CoreCompile" 
+          Condition="'$(OS)'=='Windows_NT'"
+          Inputs="$(ContractOutputPath)/$(MSBuildProjectName).dll"
+          Outputs="$(IntermediateOutputPath)$(MSBuildProjectName).ref.obj">
```

Comment:
nit, I'd call this `res.obj` or `ref.res.obj` to indicate its a resource object.  Add a comment about why its conditioned for only windows.

Reply:
Will do